colcon_core/command.py

```diff
@@ -148,6 +148,7 @@ def _main(*, command_name, argv):
     # error: no verb provided
     if args.verb_name is None:
         print(parser.format_usage())
+        print(_RTFM_TEXT)
```

Comment:
I am not sure this hint makes sense here. I would be concerned that users follow this link instead of trying `colcon --help`.